lms/templates/js/discussion/discussion-home.underscore

```diff
@@ -0,0 +1,59 @@
+<div class="discussion-article blank-slate">
+  <section class="home-header">
+    <span class="label"><%- gettext("DISCUSSION HOME:") %></span>
+    <% if (window.course_name) { %>
+      <h1 class="home-title"><%- window.course_name %></h1>
+    <% } %>
+  </section>
+
+  <% if (window.ENABLE_DISCUSSION_HOME_PANEL) { %>
+    <span class="label label-settings">
+      <%- gettext("How to use {platform_name} discussions").replace("{platform_name}", window.PLATFORM_NAME) %>
```

Comment:
I just went through the code base, and ``interpolate`` is used in both LMS and Studio There's also an LMS-only function ``interpolate_text`` which is used purely in the LMS and which seems to be almost identical but has a slightly cleaner syntax.

    // Translators: 'processor' is the name of a third-party payment processing vendor (example: "PayPal")
    return interpolate_text(gettext('Checkout with {processor}'), {processor: processorName});

In this instance, I'd use ``interpolate_text`` since it has the same syntax, is more concise, and has precedence in the code base.